### PR TITLE
refactor!: change set/get TokenId function name in LSP8 and `LSP8TokenIdSchema` to `LSP8TokenIdFormat`

### PR DESCRIPTION
**Pull request by YamenMerhi**
# What does this PR introduce?

## ♻️ Refactor

- Change `LSP8TokenIdSchema` to `LSP8TokenIdFormat`
- Change set/getTokenIdData/Batch to `set/getData|Batch|ForTokenId`

### PR Checklist

- [ ] Wrote Tests
- [ ] Wrote & Generated Documentation (readme/natspec/dodoc)
- [ ] Ran `npm run lint` && `npm run lint:solidity` (solhint)
- [ ] Ran `npm run format` (prettier)
- [ ] Ran `npm run build`
- [ ] Ran `npm run test`

### CODE REVIEW

**Review by b00ste — APPROVED** [review]

File: constants.ts [hunk 1]
```diff
@@ -410,7 +410,7 @@ export const LSP4_TOKEN_TYPES = {
 
 /**
  * @dev list of LSP8 Token ID Schemas that can be used to create different types of NFTs and represent each NFT identifiers differently.
- * @see for details see: https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-8-IdentifiableDigitalAsset.md#lsp8tokenidschema
+ * @see for details see: https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-8-IdentifiableDigitalAsset.md#lsp8tokenidformat
```
Comment (CJ42) [inline constants.ts:413]: Super good catch 👌🏻

File: contracts/LSP8IdentifiableDigitalAsset/ILSP8IdentifiableDigitalAsset.sol [hunk 2]
```diff
@@ -134,7 +134,7 @@ interface ILSP8IdentifiableDigitalAsset is IERC165, IERC725Y {
      * @param dataValue An array of values to set for the given data keys.
      * @custom:events {TokenIdDataChanged} event for each pair.
      */
-    function setTokenIdDataBatch(
+    function setDataBatchForTokenIds(
         bytes32[] memory tokenId,
```
Comment (CJ42) [inline contracts/LSP8IdentifiableDigitalAsset/ILSP8IdentifiableDigitalAsset.sol:138]: All params for the batch functions should be plural as we agreed 

bytes32[] memory tokenIds

**Review by CJ42 — CHANGES_REQUESTED** [review]
Added review comments

File: constants.ts [hunk 3]
```diff
@@ -410,7 +410,7 @@ export const LSP4_TOKEN_TYPES = {
...
  */
 export const LSP8_TOKEN_ID_SCHEMA = {
```
Comment (CJ42) [inline constants.ts:415]: The constant name needs to be changed too

**Review by CJ42 — CHANGES_REQUESTED** [review]
We need to change the constant name in constants.ts

It is still called LSP8_TOKEN_ID_SCHEMA

**Review by CJ42 — APPROVED** [review]

**Pull request merged**